operator/v1/types_console.go

```diff
@@ -39,6 +43,17 @@ type ConsoleCustomization struct {
 	// documentation URL.
 	// Invalid value will prevent a console rollout.
 	DocumentationBaseURL string `json:"documentationBaseURL,omitempty"`
+	//customProductName is the customizable product name that can be set for the web console
+	// +optional
+	CustomProductName string `json:"customProductName"`
+	//customLogoFile references a ConfigMap containing a custom product logo that can be set for the web console
+	// You only have to set this if you have your own custom product logo you wish to show on the console.
+	// The ConfigMap must exist in the openshift-config namespace and contain the following required fields:
+	// - ConfigMap.binaryData["filename"] - logo in binary form.
+	// This is best generated using: oc create configmap my-branding-config --from-file=/path/to/file -n openshift-config
+	// Image size must be less than 1 MB due to constraints on configmap size
+	// +optional
```

Comment:
Maybe

```go
// customLogoFile replaces the default OpenShift logo in the masthead and about dialog. It is a reference to a
// ConfigMap in the openshift-config namespace. This can be created with a command like
// 'oc create configmap custom-logo --from-file=/path/to/file -n openshift-config'.
// Image size must be less than 1 MB due to constraints on the ConfigMap size.
// +optional
```

Reply:
We might want to give guidance on recommended dimensions (although I don't know what those are offhand).

Reply:
It looks like we set the viewbox to 100x100 on the SVGs.  Should I add that?

Reply:
@spadgett 
`// Recommended logo is of type svg or logo size of 100px x 100px`

Reply:
100x100 can't be right since the logo is not square.

Maybe we simply say "An SVG logo is recommended, but other image types are supported." and leave it at that.

Reply:
Seems like we should indicate a recommended size, along with `svg` so its scalable.  Our own logos:

https://github.com/openshift/console/blob/master/frontend/public/imgs/openshift-dedicated-logo.svg
https://github.com/openshift/console/blob/master/frontend/public/imgs/openshift-online-logo.svg

Are `40px` high, but width is variable, roughly `260px` in browser.

Reply:
@sg00dwin

Reply:
@zherman0 the PatternFly variable `--pf-c-page__header-brand-link--c-brand--MaxHeight` sets a `max-height: 3.75rem` which is approximately 68px. So if the image is larger than 68px it will scale down proportionally. 

I would say 
```
// Recommended logo specifications: 
// Dimensions: Max height of 68px and max width of 200px
// SVG format preferred
```